lightning/src/ln/channelmanager.rs

```diff
@@ -173,6 +173,7 @@ struct ClaimableHTLC {
 	value: u64,
 	onion_payload: OnionPayload,
 	timer_ticks: u8,
+	total_msat: u64,
```

Comment:
Could be nice to document the difference between `value` and `total_msat`

Reply:
Agreed; in fact, would be necessary imo

Reply:
Is this close to accurate? 
```
/// HTLCs that are to us and can be failed/claimed by the user
 struct ClaimableHTLC {
        prev_hop: HTLCPreviousHopData,
        cltv_expiry: u32,
       /// The amount (in msats) of this MPP part
        value: u64,
        onion_payload: OnionPayload,
        timer_ticks: u8,
       /// The sum total of all MPP parts
        total_msat: u64,
 }
```

It was not super clear to me, but I am going to walk through the code more and try to figure it out.

Reply:
That looks good to me!